Tasks/SqlAzureDacpacDeployment/Utility.ps1

```diff
@@ -9,56 +9,43 @@ function Check-ServerName
     }
 }
 
-function Get-AgentStartIPAddress
+function Get-AgentIPRange
 {
-    $endpoint = (Get-VstsEndpoint -Name SystemVssConnection -Require)
-    $vssCredential = [string]$endpoint.auth.parameters.AccessToken
-
-    $vssUri = $env:SYSTEM_TEAMFOUNDATIONCOLLECTIONURI
-    if ($vssUri.IndexOf("visualstudio.com", [System.StringComparison]::OrdinalIgnoreCase) -ne -1) {
-        # This hack finds the DTL uri for a hosted account. Note we can't support devfabric since the
-        # there subdomain is not used for DTL endpoint
-        $vssUri = $vssUri.Replace("visualstudio.com", "vsdtl.visualstudio.com")
-    }
+    param(
+        [String] $serverName,
+        [String] $sqlUserName,
+        [String] $sqlPassword
+    )
 
-    Write-Verbose "Querying VSTS uri '$vssUri' to get external ip address"
+    [hashtable] $IPRange = @{}
+    $sqlCmdArgs = "-S $serverName -U $sqlUsername -P $sqlPassword -Q `"select getdate()`""
 
-    # Getting start ip address from dtl service
-    Write-Verbose "Getting external ip address by making call to dtl service"
-    $vssUri = $vssUri + "/_apis/vslabs/ipaddress"
-    $username = ""
-    $password = $vssCredential
+    Write-Verbose "Executing sqlcmd.exe with arguments $sqlCmdArgs"
 
-    $basicAuth = ("{0}:{1}" -f $username, $password)
-    $basicAuth = [System.Text.Encoding]::UTF8.GetBytes($basicAuth)
-    $basicAuth = [System.Convert]::ToBase64String($basicAuth)
-    $headers = @{Authorization=("Basic {0}" -f $basicAuth)}
+    $ErrorActionPreference = 'Continue' 
+    ( Invoke-Expression "&sqlcmd.exe --% $sqlCmdArgs" -ErrorVariable errors -OutVariable output 2>&1 ) | Out-Null
+    $ErrorActionPreference = 'Stop'
 
-    $response = Invoke-RestMethod -Uri $($vssUri) -headers $headers -Method Get -ContentType "application/json"
-    Write-Verbose "Response: $response"
+    if($errors.Count -gt 0)
+    {
+        $errorMsg = $errors[0].Exception.Message
```

Comment:
Instead of first line only, should we check for IP in complete error message. As error message might get changed.